.eslintrc

```diff
@@ -9,10 +13,24 @@
   ],
   "plugins": [
     "flowtype",
-    "prettier"
+    "prettier",
+    "jest"
   ],
   "rules": {
-    "prettier/prettier": "error",
+    "prettier/prettier": [
```

Comment:
nope, use `.prettierrc`. also, the only rules that make sense configuring are the ones already present

Reply:
Reverted